### PR TITLE
Added New Masteries, Items and some Passives's Damages, Added New DamageReduction class. Fixed AttackPlant Option, Added IsValidAttackableUnit check.

### PR DESCRIPTION
**Pull request by ExoryXD**

### CODE REVIEW

File: Aimtec.SDK/Aimtec.SDK.csproj [hunk 1]
```diff
@@ -33,13 +33,13 @@
   <ItemGroup>
     <Reference Include="Aimtec, Version=1.0.6382.1399, Culture=neutral, processorArchitecture=x86">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>$(LOCALAPPDATA)\AimtecLoader\Data\System\Aimtec.dll</HintPath>
+      <HintPath>C:\Users\Enrico\AppData\Local\AimtecLoader\Data\System\Aimtec.dll</HintPath>
```
Comment (PixlTec) [inline Aimtec.SDK/Aimtec.SDK.csproj]: This path should have been correct, why was it changed?

File: Aimtec.SDK/Aimtec.SDK.csproj [hunk 2]
```diff
@@ -33,13 +33,13 @@
...
     </Reference>
     <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\packages\Newtonsoft.Json.10.0.2\lib\net45\Newtonsoft.Json.dll</HintPath>
+      <HintPath>C:\Users\Enrico\AppData\Local\AimtecLoader\Data\System\Newtonsoft.Json.dll</HintPath>
     </Reference>
     <Reference Include="NLog, Version=4.0.0.0, Culture=neutral, PublicKeyToken=5120e14c03d0593c, processorArchitecture=MSIL">
-      <HintPath>..\packages\NLog.4.4.10\lib\net45\NLog.dll</HintPath>
+      <HintPath>C:\Users\Enrico\AppData\Local\AimtecLoader\Data\System\NLog.dll</HintPath>
```
Comment (PixlTec) [inline Aimtec.SDK/Aimtec.SDK.csproj]: This also should be changed back.

File: Aimtec.SDK/Damage/DamageItem.cs [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+                this.MagicalDamage = magicalDamage;
+                this.TrueDamage = trueDamage;
+            }
+
+            public double PhysicalDamage { get; set; }
+            public double MagicalDamage { get; set; }
+            public double TrueDamage { get; set; }
+        }
+
+        public class Item
```
Comment (PixlTec) [inline Aimtec.SDK/Damage/DamageItem.cs]: This name might get confusing for developers trying to create an `Item`. (Not for damage.) Fine for now.

Comment (LHS1140b) [inline Aimtec.SDK/Damage/DamageItem.cs]: Change!

Comment (PixlTec) [inline Aimtec.SDK/Damage/DamageItem.cs]: Its fine, it will be `DamageItem.Item` anyway.

**Pull request merged**